dataset/src/test/scala/frameless/CollectTests.scala

```diff
@@ -54,6 +54,15 @@ class CollectTests extends TypedDatasetSuite {
     check(forAll(prop[Option[SQLTimestamp]] _))
 
     check(forAll(prop[Vector[Int]] _))
+    check(forAll(prop[List[Int]] _))
+    check(forAll(prop[Seq[Int]] _))
+    check(forAll(prop[Vector[Char]] _))
+    check(forAll(prop[List[Char]] _))
+    check(forAll(prop[Seq[Char]] _))
```

Comment:
Did these collection[Char] tests used to fail?

Reply:
Yes, IIRC